### PR TITLE
[13.0] [MIG] report_qweb_parameter: Migration to v13.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT26080

please @victoralmau @pedrobaeza, can you review this?

### CODE REVIEW

File: report_qweb_parameter/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+{
+    "name": "Report QWeb Parameter",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "summary": """
+        Add new parameters for qweb templates in order to reduce field length
+        and check minimal length
+    """,
+    "author": "Creu Blanca," "Odoo Community Association (OCA)",
```
Comment (victoralmau) [inline report_qweb_parameter/__manifest__.py]:
```suggestion
    "author": "Creu Blanca," "Tecnativa", "Odoo Community Association (OCA)",
```

**Review by victoralmau — APPROVED** [review]

**Comment by CarlosRoca13** [comment]
@victoralmau Change Done!

File: report_qweb_parameter/__manifest__.py [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
     "name": "Report QWeb Parameter",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "license": "AGPL-3",
     "summary": """
         Add new parameters for qweb templates in order to reduce field length
         and check minimal length
     """,
-    "author": "Creu Blanca," "Odoo Community Association (OCA)",
+    "author": "Creu Blanca, Tecnativa, Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline report_qweb_parameter/__manifest__.py]: We haven't made merits for being co-author, so please remove it.

File: report_qweb_parameter/models/ir_qweb.py [hunk 3]
```diff
@@ -30,9 +30,9 @@ def _compile_directive_esc(self, el, options):
                 + ")"
             )
         if "t-length" in el.attrib:
-            length = el.attrib.pop("t-length")
+            length = el.attrib.pop("t-length")  # pylint: disable=attribute-deprecated
```
Comment (pedrobaeza) [inline report_qweb_parameter/models/ir_qweb.py]: Why not call the Python variable `tlength` or similar instead?

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
@pedrobaeza Changes done!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**